### PR TITLE
5-10 min ka kaam kiya.....dekhle

### PR DESCRIPTION
**Pull request by adiityasiingh**

### CODE REVIEW

File: src/components/Settings.js [hunk 1]
```diff
@@ -56,22 +56,22 @@ function Settings() {
       return (
         <div className="row">
           <div className="col-12 py-3 fw-bold">
-            <i className="bi bi-emoji-smile mx-2"></i> You are online
+            <i class="bi bi-person-circle"></i> You are Logged in<i class="bi bi-wifi"></i>
```
Comment (Vishesh-Pandey) [inline src/components/Settings.js]: make sure to use className instead of class.

File: src/components/Settings.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ function Settings() {
       return (
         <div className="row">
           <div className="col-12 py-3 fw-bold">
-            <i className="bi bi-emoji-frown mx-2"></i>You are Offline
+            <i class="bi bi-person-circle"></i>You are Offline<i class="bi bi-wifi-off"></i>
```
Comment (Vishesh-Pandey) [inline src/components/Settings.js]: class -> className

File: src/features/websites/Websites.js [hunk 3]
```diff
@@ -137,13 +137,13 @@ function Websites() {
...
-        <div className="col-2">
+        <div className="col-2 py-3">
           <img
             data-bs-toggle="modal"
             data-bs-target="#exampleModal"
-            className="w-100 rounded-3 border border-1 border-secondary"
+            className="w-100 rounded-3  border-1 border-secondary"
             style={{ cursor: "pointer" }}
-            src="https://static.thenounproject.com/png/768833-200.png"
+            src="https://cdn.pixabay.com/photo/2017/03/19/03/51/material-icon-2155448_640.png"
```
Comment (Vishesh-Pandey) [inline src/features/websites/Websites.js:146]: using the square image is highly recommended to maintain the proper ratio.

**Review by Vishesh-Pandey — COMMENTED** [review]
update class to className and use a square image.

File: src/features/websites/Websites.js (hunk 3 above)
Comment (adiityasiingh) [reply src/features/websites/Websites.js:146]: image square hi h bas bg nae h usme.....i think
correct me if i'm wrong

Comment (Vishesh-Pandey) [reply src/features/websites/Websites.js:146]: image

check the image on a new tab. it seems it has more width than height.

**Review by Vishesh-Pandey — APPROVED** [review]

**Pull request merged**